arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/predef-test.kt

```diff
@@ -111,13 +110,9 @@ fun <L, R> Arb.Companion.either(left: Arb<L>, right: Arb<R>): Arb<Either<L, R>>
   return Arb.choice(failure, success)
 }
 
-fun Arb.Companion.intRange(min: Int = 0, max: Int = 1000): Arb<IntRange> =
-  Arb.int(min, max).flatMap { a ->
-    Arb.int(min, max).map { b ->
-      val first = min(a, b)
-      val last = max(a, b)
-      first..last
-    }
+fun Arb.Companion.intRange(min: Int = Int.MIN_VALUE, max: Int = Int.MAX_VALUE): Arb<IntRange> =
```

Comment:
@nomisRev , thank you so much for making it work for Kotest `4.1.0` :tada: Just a small question, could this change impact on current tests? I mean, the max value changed from 1000 to 2147483647. Thanks in advance!!

Reply:
And the min value also changes (from 0 to -2147483648).

Reply:
Hey @rachelcarmena,
The reason for this change was to make the generator more accurate.
The only current usage from this `Arb` explicitly defines `min` & `max`, so it doesn't impact the current suite.

Reply:
Great, thank you so much Simon!! :raised_hands: